config.py

```
@@ -5,6 +5,37 @@
 modifying the main pipeline code.
 """
 
+import torch
+import subprocess
```

Ruff (0.11.9)

`subprocess` imported but unused

Remove unused import: `subprocess`

(F401)